code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1235,3 +1235,83 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "glass_yellow"
 	glass_name = "Eggnog"
 	glass_desc = "For enjoying the most wonderful time of the year."
+
+// IPC ALCHOHOL WOOOOOOOOOOO
+/datum/reagent/consumable/ethanol/synthanol
+	name = "Synthanol"
+	id = "synthanol"
+	description = "A runny liquid with conductive capacities. Its effects on synthetics are similar to those of alcohol on organics."
```

Comment:
... is it runny? or a liquid?

One infers complete fluidity, the other one gloopy.

Reply:
This description is only shown when you put the reagent in a reagent scanner, which I dont think ANYONE does with bar drinks.